lib/game.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Game
+  attr_writer :cell
+
+  def initialize(cell)
+    @cell = cell
+  end
+
+  def win_game
+    diagonal || vertical || horizontal
+  end
+
+  private
+
+  def diagonal
+    left_diagonal || right_diagonal
+  end
+
+  def left_diagonal
+    @cell[0][0] == @cell[1][1] && @cell[1][1] == @cell[2][2]
+  end
+
+  def right_diagonal
+    @cell[2][0] == @cell[1][1] && @cell[1][1] == @cell[0][2]
+  end
+
+  def vertical
+    vertical_one || vertical_two || vertical_three
+  end
+
+  def vertical_one
+    @cell[0][0] == @cell[1][0] && @cell[1][0] == @cell[2][0]
+  end
+
+  def vertical_two
+    @cell[0][1] == @cell[1][1] && @cell[1][1] == @cell[2][1]
+  end
+
+  def vertical_three
+    @cell[0][2] == @cell[1][2] && @cell[1][2] == @cell[2][2]
+  end
+
+  def horizontal
+    horizontal_one || horizontal_two || horizontal_three
+  end
+
+  def horizontal_one
+    @cell[0][0] == @cell[0][1] && @cell[0][1] == @cell[0][2]
+  end
+
+  def horizontal_two
+    @cell[1][0] == @cell[1][1] && @cell[1][1] == @cell[1][2]
+  end
```

Comment:
This example was just to ignite some ideas and will not work as-is. Currently, your code breaks the Dry principle (don't repeat your self). There are more efficient solutions with less repetition.  Hint!! you can use ranges also eg: [1..3]